level shifter component defined and added to worklib folder for ease of updating

oxfpete opened the pull request:
Not stored in usual library area (we tend to use a folder called mylib to save confusion) but in the worklib folder is the sn74axc8t245rhlr level shifter that will compile and use the footprint supplied to me (RHL0024A.dra)

DavidCussans commented:
Pete,

Ah. I see I should have found time to check the schematics earlier.... I have just merged the pull request ( I had some issues because I had been careless enough to accidentally check in some temporary files (junk) which caused a clash with yours )

The sfp_los and sfp_fault lines from the SFPs are 3.3V signals. They need to be level shifted to​2.5V ( P2V5 )

The sfp_tx_disable lines into the SFP are 3.3V signals. They need to be level shifted  from​ 2.5V

.... so there should be another three octal level shifters on the level shifters page .

Apologies....

David

Dr David Cussans                    University of Bristol,
office: +44-(117)-95 46879   H.H. Wills Physics Laboratory,
     lab: +44-(117)-33 17199   Tyndall Avenue,
     fax: +44-(117)-925 5624   Bristol BS8 1TL, UK
________________________________
From: Peter Hastings ***@***.***>
Sent: Thursday, May 16, 2024 11:22 AM
To: uob-hep-cad/uob-hep-pc065 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [uob-hep-cad/uob-hep-pc065] level shifter component defined and added to worklib folder for ease of updating (PR #38)


Closed #38<https://github.com/uob-hep-cad/uob-hep-pc065/pull/38>.

—
Reply to this email directly, view it on GitHub<https://github.com/uob-hep-cad/uob-hep-pc065/pull/38#event-12830094011>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABLNCKRNBFIS5XZSBSRDQTTZCR3GPAVCNFSM6AAAAABHTT6FOSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJSHAZTAMBZGQYDCMI>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

oxfpete commented:
No problem. It might be sometime tomorrow before I send another update!

From: David Cussans ***@***.***>
Sent: 16 May 2024 16:00
To: uob-hep-cad/uob-hep-pc065 ***@***.***>
Cc: Peter Hastings ***@***.***>; State change ***@***.***>
Subject: Re: [uob-hep-cad/uob-hep-pc065] level shifter component defined and added to worklib folder for ease of updating (PR #38)

Pete,

Ah. I see I should have found time to check the schematics earlier.... I have just merged the pull request ( I had some issues because I had been careless enough to accidentally check in some temporary files (junk) which caused a clash with yours )

The sfp_los and sfp_fault lines from the SFPs are 3.3V signals. They need to be level shifted to​2.5V ( P2V5 )

The sfp_tx_disable lines into the SFP are 3.3V signals. They need to be level shifted from​ 2.5V

.... so there should be another three octal level shifters on the level shifters page .

Apologies....

David

Dr David Cussans University of Bristol,
office: +44-(117)-95 46879 H.H. Wills Physics Laboratory,
lab: +44-(117)-33 17199 Tyndall Avenue,
fax: +44-(117)-925 5624 Bristol BS8 1TL, UK
________________________________
From: Peter Hastings ***@***.***<mailto:***@***.***>>
Sent: Thursday, May 16, 2024 11:22 AM
To: uob-hep-cad/uob-hep-pc065 ***@***.***<mailto:***@***.***>>
Cc: Subscribed ***@***.***<mailto:***@***.***>>
Subject: Re: [uob-hep-cad/uob-hep-pc065] level shifter component defined and added to worklib folder for ease of updating (PR #38)


Closed #38<https://github.com/uob-hep-cad/uob-hep-pc065/pull/38>.

—
Reply to this email directly, view it on GitHub<https://github.com/uob-hep-cad/uob-hep-pc065/pull/38#event-12830094011>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABLNCKRNBFIS5XZSBSRDQTTZCR3GPAVCNFSM6AAAAABHTT6FOSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJSHAZTAMBZGQYDCMI>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***<mailto:***@***.***>>


—
Reply to this email directly, view it on GitHub<https://github.com/uob-hep-cad/uob-hep-pc065/pull/38#issuecomment-2115489507>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BEU4WDCTWUESOBA4HGFN66TZCTCXZAVCNFSM6AAAAABHTT6FOSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMJVGQ4DSNJQG4>.
You are receiving this because you modified the open/close state.Message ID: ***@***.******@***.***>>
